package.json

```diff
@@ -7,9 +7,10 @@
     "test": "test"
   },
   "dependencies": {
+    "jquery": "^1.9.1",
+    "npm": "^6.0.1",
```

Comment:
did you need to add the npm requirement for nuget?

Reply:
no, that is a mistake, thanks, I will fix it.